routes/web.php

```diff
@@ -72,3 +88,23 @@
 
     Route::get('/secretariat/users', 'SecretariatController@list')->name('secretariat.users');
 });
+
+//test emails with urls
+Route::middleware(['auth', 'log'])->group(function () {
+    Route::get('/test_mails/{mail}/{send?}', function ($mail, $send = false) {
```

Comment:
Same here, move this to a controller.

Reply:
Should I create a new Controller just for these?

Reply:
Yeah, we will probably extend it later.